libs/@local/hash-authorization/Cargo.toml

```diff
@@ -27,4 +28,3 @@ tokio = { version = "1.33.0", default-features = false , features = ["macros", "
 
 [features]
 utoipa = ["dep:utoipa"]
-
```

Comment:
intended?

Reply:
Yes, I found the setting to remove trailing blank linkes (but keeping one) again 😄